wordpress-mobile/WordPress-iOS
# Remove feature flag for Recommend App to Others

Fixes n/a

Removes the feature flag for the Recommend App feature. This cleans up the conditionals in Me and About screen. Additionally, note that the About screen, the `isRecommendAppSectionEnabled` is still used since the feature is hidden for Jetpack app.

## To test

- Verify that the Recommend App feature works in the Me screen.
- Verify that the Recommend App feature works in the About screen.

## Regression Notes
1. Potential unintended areas of impact
Should be none.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested the feature.

3. What automated tests I added (or what prevented me from doing so)
n/a.

PR submission checklist:
